app/repositories/SubscriptionDataRepository.scala

```diff
@@ -98,17 +98,7 @@ class SubscriptionDataRepository @Inject()(mongo: ReactiveMongoComponent,
   )
 
   collection.indexesManager.ensure(sessionIdIndex)
-
-  collection.indexesManager.drop(ttlIndex.name.get) onComplete {
-    _ => collection.indexesManager.ensure(ttlIndex)
+  collection.indexesManager.ensure(ttlIndex)
   }
```

Comment:
This bracket should also be removed

Reply:
@AlexRimmerHMRC Done

Reply:
Still not seeing this as done, the bracket is still in this PR